### DISCUSSION TITLE
Help please: How to get icloudpd > 1.8.1 running in Synology Container Manager?

### DISCUSSION THREAD

**Discussion by cmdux** [icloud-photos-downloader/icloud_photos_downloader#728] [Q&A]
Hi there,
I want to use icloudpd to backup my iCloud photos to my Synology NAS. I managed to set up a Docker container in Synology Container Manager with this manual:

https://www.reddit.com/r/synology/comments/wdwfb9/comment/iixs00x/?context=1

Unfortunately this manual just works up for version 1.8.1. 

I would like to use the latest version of icloudpd because I have a shared library that I also want to backup. 

When I try to set up the container in Synology Container Manager with the same settings it just fails after some seconds.

I don't know what I'm doing wrong and searched the internet without luck. Some advice would be really nice. Thanks a lot!

**Comment by AndreyNikiforov** [comment]
From the first glance there is nothing in that manual that should not work for the latest release of the `icloudpd`. What problems are you facing?

Two notes:
- I don't have Synology and never had
- #729 suggests that auth broken for all users/versions, suspecting changes on Apple side

**Comment by sentience** [comment]
I've got it working on my Synology NAS in Container Manager. Let me know if you still need help and I can write up some instructions.
